node_modules/gh-series/tests/test-patterns.js

```diff
@@ -0,0 +1,259 @@
+                    SeriesTestUtil.assertGetSerie(global.tests.admins.cam2014.client, serie.id, null, function(serie) {
+                        assert.strictEqual(serie.metadata.pattern, 'Mi3 Tu 14; Mi4-5 Tu 13');
+
+                        // Make the event a 2 hour session
+                        end = moment(end).add(1, 'hour').format();
+                        EventsTestUtil.assertUpdateEvent(global.tests.admins.cam2014.client, eventA.id, {'end': end}, function() {
+                            SeriesTestUtil.assertGetSerie(global.tests.admins.cam2014.client, serie.id, null, function(serie) {
+                                assert.strictEqual(serie.metadata.pattern, 'Mi3 Tu 14-16; Mi4-5 Tu 13');
+
+                                // Make the event a 3 hour session by letting it start at 12 o'clock
+                                start = moment(start).subtract(1, 'hour').format();
+                                EventsTestUtil.assertUpdateEvent(global.tests.admins.cam2014.client, eventA.id, {'start': start}, function() {
+                                    SeriesTestUtil.assertGetSerie(global.tests.admins.cam2014.client, serie.id, null, function(serie) {
+                                        assert.strictEqual(serie.metadata.pattern, 'Mi3 Tu 13-16; Mi4-5 Tu 13');
+                                        return callback();
+                                    });
+                                });
+                            });
+                        });
+                    });
+                });
+            });
+        });
+
+        /**
+         * Test that verifies that organisers who are linked users are included when aggregating event organisers
+         */
+        it('verify organisers who are linked users are included when aggregating event organisers', function(callback) {
+            // Create a serie with 3 events that can aggregate
+            setupAggregation(function(eventA, eventB, eventC, serie) {
+
+                // Create an event, but don't specify an organiser. This will make it default
+                // to the current user. Ensure the aggregator can handle this
+                EventsTestUtil.assertCreateEvent(global.tests.admins.cam2014.client, 'A', '2014-11-11T20:00:00', '2014-11-20T14:00:00', {'location': 'Room 2'}, function(eventD) {
+                    SeriesTestUtil.assertAddSeriesEvents(global.tests.admins.cam2014.client, serie.id, [eventD.id], function() {
+
+                        // Get the serie and check if the aggregated metadata is correct
+                        SeriesTestUtil.assertGetSerie(global.tests.admins.cam2014.client, serie.id, null, function(serie) {
+                            assert.strictEqual(serie.metadata.organisers.length, 3);
+                            return callback();
+                        });
+                    });
+                });
+            });
+        });
+    });
+
+    describe('Patterns', function() {
+
```

Comment:
I wonder if it would be worth adding a TODO here that states that we can load the 2013-14 timetable data once we have import and then check the summaries for everything in there. That seems like the best way of validating that the pattern generation works.